### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by jeannekamikaze**
- Remove quotes from cross-reference for consistency with other cross-references.
- Pass ... to.

### CODE REVIEW

File: documentation/content/en/books/handbook/basics/_index.adoc [hunk 1]
```diff
@@ -666,7 +666,7 @@ In FreeBSD, every file and directory has an associated set of permissions and se
 Understanding how permissions work is necessary to make sure that users are able to access the files that they need and are unable to improperly access the files used by the operating system or owned by other users.
 
 This section discusses the traditional UNIX(R) permissions used in FreeBSD.
-For finer grained file system access control, refer to crossref:security[fs-acl,“Access Control Lists”].
+For finer grained file system access control, refer to crossref:security[fs-acl,Access Control Lists].
```
Comment (pauamma) [inline documentation/content/en/books/handbook/basics/_index.adoc:669]: "finer-grained", while here.

File: documentation/content/en/books/handbook/basics/_index.adoc [hunk 2]
```diff
@@ -717,8 +717,8 @@ When reading the "Directory Listing" column, a `-` is used to represent a permis
 |`rwx`
 |===
 
-Use the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
-For example, an `ls -l` in an arbitrary directory may show:
+Pass the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/basics/_index.adoc:720]: I'd keep "use" here as more natural for an end-user audience (as opposed to developer audience).

**Review by pauamma — CHANGES_REQUESTED** [review]
Again, thanks for these.

File: documentation/content/en/books/handbook/basics/_index.adoc (hunk 1 above)
Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/basics/_index.adoc]: Done, thank you.

File: documentation/content/en/books/handbook/basics/_index.adoc [hunk 3]
```diff
@@ -717,7 +717,7 @@ When reading the "Directory Listing" column, a `-` is used to represent a permis
 |`rwx`
 |===
 
-Pass the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
+Use the `-l` argument when invoking man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/basics/_index.adoc:720]: That works but feels a bit stilted. How about "use... with..."?

Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/basics/_index.adoc]: Much better, thanks.

**Review by pauamma — APPROVED** [review]

**Comment by dbaio** [comment]
Committed on a1b6b175b24ebacf5ac3e7de8ea3c15bb6148be5. Thank you.

**Pull request closed without merging**